Vagrantfile

```diff
@@ -7,6 +7,7 @@ Vagrant.configure(2) do |config|
         d.has_ssh = true
         d.build_args = ['--build-arg', 'userid=' + `id -u`.strip, '--build-arg', 'groupid=' + `id -g`.strip]
         d.create_args = ['--cap-add=NET_ADMIN', '--device=/dev/net/tun']
+        d.create_args = ['--security-opt', 'seccomp:unconfined']
```

Comment:
Write details in commit message instead of referring to a link that you have no control over? Maybe even leave a comment in Vagrantfile?

Reply:
Can of course keep link as well, but not only link. Might be gone tomorrow because SO changed their db or something for all we know.